README.md

```diff
@@ -92,14 +92,17 @@ You can use
 
 ## JAR 
 
-* Download the [JAR file](https://github.com/zeebe-io/zeebe-http-worker/releases) 
+* Build the JAR file 
```

Comment:
I could imagine the maven build as an addition - but why do you replace the download option with a build only?

Reply:
the original link points to a different project - `zeebe-http-worker`, this project has no releases so there's no JAR we can link to